### PR TITLE
add bone marrow microvessel

### LINKED ISSUE #2945: NTR: Bone marrow microvessel
**Preferred term label:**
Bone marrow microvessel

**Definition (free text, please give PubMed ID)**
Microvessel located in the bone marrow, including arterioles and sinusoids. (PMID:32286470)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
is_a blood microvessel, UBERON_8410081
part_of stroma of bone marrow UBERON_0007195

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2945

### CODE REVIEW

**Comment by anitacaron** [comment]
We're running an Uberon release, please hold this PR until new release is out.

**Comment by paulacalipho** [comment]
@anitacaron OK

**Review by bvarner-ebi — COMMENTED** [review]
Please see inline comment.

Interestingly, based on the current axioms, it is inferred that this structure is a subclass of 'bone foramen', which does not seem accurate.

Screenshot 2023-07-19 at 14 33 38

Investigating this is probably outside the scope of this ticket and should be investigated in a separate ticket.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224155,6 +224155,15 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0001137 ! dorsum
 property_value: dcterms-date "2023-06-23T09:06:25Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480067
+name: bone marrow microvessel
+def: "Microvessel located in the bone marrow, including arterioles and sinusoids." [PMID:32286470]
+is_a: UBERON:8410081 ! blood microvessel
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0007195 ! stroma of bone marrow
```
[the remark this answers is not in the document]
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224164]: @bvarner-ebi I changed to 'bone marrow'

**Review by bvarner-ebi — APPROVED** [review]

**Comment by bvarner-ebi** [comment]
> Please see inline comment.
> 
> Interestingly, based on the current axioms, it is inferred that this structure is a subclass of 'bone foramen', which does not seem accurate.
> 
> Investigating this is probably outside the scope of this ticket and should be investigated in a separate ticket.

If `‘part of' some 'bone marrow’` is changed to`'located in' some 'bone marrow’`

then `SubClass Of some ‘bone foramen’` is no longer inferred. It also matches the text definition.

**Comment by paulacalipho** [comment]
@bvarner-ebi OK, I understand that I should change to 'located in' some 'bone marrow’. I already merged and deleted the branch. Can I do a new PR without an issue? or should I create a new issue and then do the new PR?

**Comment by bvarner-ebi** [comment]
@paulacalipho, you can do a new PR for this change and reference this PR without a new issue. 
Apologies for not making this suggestion before approval/merge.

**Comment by paulacalipho** [comment]
@bvarner-ebi no worries! I will do that

**Pull request merged**